datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/recovery/SdxUpgradeRecoveryService.java

```diff
@@ -64,9 +64,10 @@ public SdxRecoverableResponse validateRecovery(SdxCluster sdxCluster, SdxRecover
             String initiatorUserCrn = ThreadBasedUserCrnProvider.getUserCrn();
             if (Objects.nonNull(request) && request.getType() == SdxRecoveryType.RECOVER_WITH_DATA) {
                 String clusterRuntime = sdxCluster.getRuntime();
-                datalakeDRProto.DatalakeBackupInfo lastSuccessfulBackup = datalakeDrClient.getLastSuccessfulBackup(sdxCluster.getClusterName(),
-                        initiatorUserCrn, Optional.of(clusterRuntime));
-                if (Objects.isNull(lastSuccessfulBackup)) {
+                Optional<datalakeDRProto.DatalakeBackupInfo> lastSuccessfulBackup = backupRestoreService.getLastSuccessfulBackupInfoWithRuntime(
+                        sdxCluster, initiatorUserCrn, clusterRuntime
+                );
+                if (lastSuccessfulBackup.isEmpty()) {
```

Comment:
Why is the change made?

Reply:
This was really just to remove the unnecessary uses of the DrClient and just use the backuprestoreservice but this isn't a necessary change.